### PR TITLE
update setup.sh for dependency libjpeg-turbo

### PR DESCRIPTION
**Pull request by Rob1in**
Tested on bullseye

Tested on Mac

Tested on linux x86_64

### CODE REVIEW

File: etc/setup.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ do_bullseye(){
 	echo "deb [signed-by=/usr/share/keyrings/nodesource.gpg] https://deb.nodesource.com/node_18.x $(grep VERSION_CODENAME /etc/os-release | cut -d= -f2) main" > /etc/apt/sources.list.d/nodesource.list
 
 	# Install most things
-	apt-get update && apt-get install -y build-essential nodejs libnlopt-dev libx264-dev libopus-dev libtensorflowlite-dev protobuf-compiler protoc-gen-grpc-web ffmpeg && apt-get clean
+	apt-get update && apt-get install -y build-essential nodejs libnlopt-dev libx264-dev libopus-dev libtensorflowlite-dev protobuf-compiler protoc-gen-grpc-web ffmpeg libjpeg62-turbo-dev && apt-get clean
```
Comment (edaniels) [inline etc/setup.sh:30]: What's the 62 here?

**Review by edaniels — APPROVED** [review]
LGTM but curious about what 62 is.

File: etc/setup.sh (hunk 1 above)
Comment (Rob1in) [reply etc/setup.sh:30]: It seems like it's referring to the version of libjpeg (6.2)

Comment (edaniels) [reply etc/setup.sh:30]: I don't see where that version exists? How'd you select it?

Comment (Rob1in) [reply etc/setup.sh:30]: Following [this](https://github.com/viamrobotics/rdk/pull/1846#issuecomment-1434902535)

**Pull request merged**